theme/templates/resource-landing-page/title-section.html

```diff
@@ -327,7 +362,62 @@ <h2 id="resource-title">{{ title }}</h2>
         .title-builder-input {
             margin-top: 1em;
         }
+        .path-align {
+          word-wrap: break-word;
+          white-space: normal !important;
+        }
     </style>
+
+    <script>
```

Comment:
this is a long enough `<script>` block that I would recommend moving it to it's own .js file and importing into `title-section.html`

Reply:
Good idea. Added a new js file.